docs/src/pages/system/styled/UsingWithSx.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {
+  styled,
+  createTheme,
+  ThemeProvider,
+  experimental_sx as sx,
+} from '@mui/system';
+
+const customTheme = createTheme({
+  palette: {
+    primary: {
+      main: '#1976d2',
+      contrastText: 'white',
+    },
+  },
+});
+
+const MyThemeComponent = styled('div')(
+  sx({
+    color: 'primary.contrastText',
+    backgroundColor: 'primary.main',
+    padding: 1,
+    borderRadius: 1,
+  }),
```

Comment:
One question, does `sx` work without ThemeProvider?

Reply:
Yes, it's just that I could not really test that the values are applied :) I will add test for it too, just to ensure it does not throw